test/e2e/windows/memory_limits.go

```diff
@@ -184,12 +188,14 @@ func getNodeMemory(f *framework.Framework) nodeMemory {
 	framework.ExpectNotEqual(nodeList.Size(), 0)
 
 	ginkgo.By("Getting memory details from node status and kubelet config")
-
 	status := nodeList.Items[0].Status
-
 	nodeName := nodeList.Items[0].ObjectMeta.Name
 
-	kubeletConfig, err := e2ekubelet.GetCurrentKubeletConfig(nodeName, f.Namespace.Name, true)
+	framework.Logf("Getting configuration details for node %s", nodeName)
+	request := f.ClientSet.CoreV1().RESTClient().Get().Resource("nodes").Name(nodeName).SubResource("proxy").Suffix("configz")
```

Comment:
is this something stable? [the current docs says](https://kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/#generating-a-file-that-contains-the-current-configuration)

> Caution: The kubelet's configz endpoint is there to help with debugging, and is not a stable part of kubelet behavior. Do not rely on the behavior of this endpoint for production scenarios or for use with automated tools.

Reply:
I'm wondering if this can add more flakiness or break in the future , but it is just a comment , not blocking anything

Reply:
good call out.  `e2ekubelet.GetCurrentKubeletConfig` was essentially doing the same thing only using kubectl as an external process to open a port so this change is not adding a dependency on that endpoint but changing the way it is called.  In my runs locally I didn't see the flake with this new approach

If in the future this proves to be unstable we can revisit the approach to getting the active configuration.